docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Practical Service Hosting
+
+### Edge Router Tunneler Hosting
+
+#### Single Application Endpoint
+When hosting services with the edge router tunneler (ER/T) combination you'll need to use a service configurations. We're going
+to start off simply, with one service endpoint and build up from there.
+
+Our application server is going to be on a local subnet at IP 192.168.3.136, port 8080. For our `test` service, we make
```

Comment:
This is an example, which is cool, but I would expect it to come after introducing a concept, and then explaining the concept, and then an example is shown. Or, it should setup the explanation. It doesn't seem to setup an explanation (below) and I don't quite understand by just reading the words (not looking at the CLI) what this is doing. I think it needs more explanation